heron/uploaders/src/java/com/twitter/heron/uploader/docker/sample.yaml

```diff
@@ -0,0 +1,11 @@
+# uploader class for transferring the topology jar/tar files to storage
+heron.class.uploader:         com.twitter.heron.uploader.docker.DockerUploader
+
+#base image
+heron.uploader.docker.base:                      ubuntu:latest
```

Comment:
yeah I think building out the full docker support wouldn't be too much more work from where this PR is now and would actually simplify how all this works.